src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public int getRadius() {
+        return radius;
+    }
+
+    public void setRadius(int radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public String draw() {
+        return "Figure " + getClass().getSimpleName() + ", area: " + getArea()
+                + " sq. units, color: " + getColor() + ", radius: " + getRadius();
```

Comment:
do you need to use method `gerRadius()` here? you have direct access 
change in every class accordingly